app/views/layouts/_beta_message.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="dgu-beta__banner">
   <div class="dgu-beta__message">
     <p>
-      We’ve been improving data.gov.uk to help you find and use open government data.
+      We’ve been improving data.gov.uk to help you find and use open government data.<br />
```

Comment:
I think there's a `text` class we can add apply instead of the break. We use it on the static pages.